README.md

```diff
@@ -109,6 +109,108 @@ nginx example (in addition to SSL setup):
     }
 ```
 
+
 # Cluster management
 
-TODO
+## Perequisites
+
+Install autossh using package manager, and "dwq" from pip3.
+
+
+## Setup
+
+1. set up ssh authentication to ci.riot-os.org.
+
+E.g., add this to `~/.ssh/config`:
+
+```
+Host murdock
+HostName ci.riot-os.org
```

Comment:
Just to clarify, if we had another murdock master running, this would be the HostName of that (eg. `HostName test-ci.riot-os.org`)?

Reply:
Yes!